components/builder-api/src/http/handlers.rs

```diff
@@ -162,6 +163,25 @@ pub fn list_user_origins(req: &mut Request) -> IronResult<Response> {
     }
 }
 
+pub fn validate_origin_invitation(conn: &mut BrokerConn,
+                                  req: &OriginInvitationAcceptRequest)
+                                  -> IronResult<bool> {
+    let mut request = OriginInvitationValidateRequest::new();
+    request.set_invite_id(req.get_invite_id());
+    request.set_account_accepting_request(req.get_account_accepting_request());
+    match conn.route::<OriginInvitationValidateRequest, OriginInvitationValidateResponse>(&request) {
```

Comment:
Boolean responses can be represented as `NetOk`, no need to define a `XResponse`. The bad case should be represented as a `NetErr`